docs/src/css/overrides.css

```diff
@@ -135,3 +135,8 @@ table thead tr {
 [class*='navbar__link']:not([class*='header-github']):not([role='button']) {
   display: none;
 }
+
+/* Remove in @swmansion-t-rex-ui 0.0.10 patch */
```

Comment:
I'm not sure if you use any sort of `TODO` extensions, but it could be useful to mark it as `TODO` 😅